packages/rollup-plugin-styles/src/index.ts

```
@@ -0,0 +1,402 @@
+                if (res.map && sourceMap) {
+                    const fileName = this.getFileName(cssFileId);
+
+                    const assetDir =
+                        typeof options_.assetFileNames === "string"
+                            ? normalizePath(path.dirname(options_.assetFileNames))
+                            : typeof options_.assetFileNames === "function"
+                              ? normalizePath(path.dirname(options_.assetFileNames(cssFile)))
+                              : "assets"; // Default for Rollup v2
+
+                    const map = mm(res.map)
+                        .modify((m) => (m.file = path.basename(fileName)))
```

Biome

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)